pymedphys/_gui/streamlit/mudensity/main.py

```diff
@@ -692,11 +691,13 @@ def main():
     default_png_output_directory = config["output"]["png_directory"]
 
     if advanced_mode:
-        """
-        ### Image record
+        st.write(
+            """
+            ### Image record
 
-        Path to save the image of the results for posterity
-        """
+            Path to save the image of the results for posterity
```

Comment:
This wording is not English as a Second language friendly.
If we internationalize and localize, it's fine.
Not critical but something to keep in mind

Reply:
Good call. I hadn't thought of that.